taichi/codegen/codegen_llvm.cpp

```diff
@@ -1571,6 +1571,13 @@ void CodeGenLLVM::create_offload_struct_for(OffloadedStmt *stmt, bool spmd) {
 
   llvm::Function *body = nullptr;
   auto leaf_block = stmt->snode;
+  // generate struct for on bit array's parent node(usually dense) instead of
+  // itself
```

Comment:
```suggestion
  // When looping over bit_arrays, we generate struct for on bit array's parent node (usually "dense")
  // instead of itself for higher performance. Also, note that the loop must be bit_vectorized.
```